src/_pytest/logging.py

```diff
@@ -183,6 +244,12 @@ def add_option_ini(option, dest, default=None, type=None, **kwargs):
         default=DEFAULT_LOG_DATE_FORMAT,
         help="log date format as used by the logging module.",
     )
+    add_option_ini(
+        "--log-auto-indent",
+        dest="log_auto_indent",
+        default=None,
+        help="Auto indent multiline messages passed to the logging module. accepts true|on, false|off or an integer",
```

Comment:
`Auto indent multiline messages passed to the logging module. Accepts true|on, false|off or an integer.`